platform_versioned_docs/version-24.3/troubleshooting_and_faqs/studios_troubleshooting.mdx

```diff
@@ -9,37 +9,37 @@ tags: [faq, help, data studios, troubleshooting]
 
 In your interactive analysis environment, open a new terminal and type `ls -la /workspace/data`. This displays all the mounted datasets available in the current studio session.
 
-![](./_images/data_studios_notebook_fusion.png)
+![](./_images/studios_notebook_fusion.png)
 
-## Data Studio session is stuck in **starting**
+## Studio session is stuck in **starting**
 
-If your studio session doesn't advance from **starting** status to **running** status within 30 minutes, and you have access to the AWS Console for your organization, check that the AWS Batch compute environment associated with the data studio session is in the **ENABLED** state with a **VALID** status. You can also check the **Compute resources** settings. Contact your organization's AWS administrator if you don't have access to the AWS Console.
+If your studio session doesn't advance from **starting** status to **running** status within 30 minutes, and you have access to the AWS Console for your organization, check that the AWS Batch compute environment associated with the Studio session is in the **ENABLED** state with a **VALID** status. You can also check the **Compute resources** settings. Contact your organization's AWS administrator if you don't have access to the AWS Console.
 
 If sufficient compute environment resources are unavailable, **Stop** the studio session and any others that may be running before trying again. If you have access to the AWS Console for your organization, you can terminate a specific session from the AWS Batch Jobs page (filtering by compute environment queue).
 
-## Data Studio session status is **errored**
+## Studio session status is **errored**
 
 The **errored** status is generally related to issues encountered when creating the studio session resources in the compute environment (e.g., invalid credentials, insufficient permissions, network issues). It can also be related to insufficient compute resources, which are set in your compute environment configuration. Contact your organization's AWS administrator if you don't have access to the AWS Console. Also contact your Seqera account executive so we can investigate the issue.
```

Comment:
```suggestion
The **errored** status is generally related to issues encountered when creating the Studio session resources in the compute environment (e.g., invalid credentials, insufficient permissions, network issues). It can also be related to insufficient compute resources, which are set in your compute environment configuration. Contact your organization's AWS administrator if you don't have access to the AWS Console. Also contact your Seqera account executive so we can investigate the issue.
```